src/store/pool/hooks.ts

```diff
@@ -355,3 +357,63 @@ export function useAPYs() {
 
   return apys;
 }
+
+export enum PairState {
```

Comment:
Please move the logic in the hooks directory like uniswap code.

Reply:
OK